.coveragerc

```diff
@@ -1,11 +1,15 @@
 [run]
 branch = True
-concurrency = multiprocessing
+concurrency =
+    multiprocessing
```

Comment:
It would actually be great to split these and the travis changes into a separate PR

Reply:
Good point. Will do.